topics/ecology/tutorials/remote-sensing/tutorial.md

```diff
@@ -0,0 +1,441 @@
+>      - {% icon param-file %} *"Input data"*: `Reflectance` (output of **Preprocessing sentinel 2 data** {% icon tool %})
+>    - {% icon param-select %} *"Do you want to do a PCA or a SPCA ?"*: 'PCA'
+>
+>
+>    > <tip-title>If you want to use your own files ENVI BIL</tip-title>
+>    >
+>    > **TODO**: *Check that the "Input raster" datatype is bil and that "Input raster header" datatype is hdr*
+>    >
+>    > * Go on your raster data
+>    > * Click on {% icon galaxy-pencil %} to edit it
+>    > * Click on {% icon galaxy-chart-select-data %} Datatypes
+>    > * On "New Type" **Select** bil
+>    > * Press **Save**
+>    >
+>    >
+>    >    > <comment-title></comment-title>
+>    >    >
+>    >    > Do the same for the raster header with the datatype hdr
+>    >    {: .comment}
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+## **Compute biodiversity indices**
+
+> <hands-on-title>Biodiversity indicators for global remote sensing data</hands-on-title>
+>
+> 1. {% tool [Compute biodiversity indices](toolshed.g2.bx.psu.edu/repos/ecology/srs_global_indices/srs_global_indices/0.0.1) %} with the following parameters:
+>    - {% icon param-select %} *"In which format are your data ?"*: 'The data you are using are in a zip folder Reflectance'
+>      - {% icon param-file %} *"Input data"*: `Reflectance` (output of **Preprocessing sentinel 2 data** {% icon tool %})
+>    - {% icon param-text %} *"Write a number of the value of alpha"*: '1'
+>
+>
+>    > <tip-title>If you want to use your own files ENVI BIL</tip-title>
+>    >
+>    > **TODO**: *Check that the "Input raster" datatype is bil and that "Input raster header" datatype is hdr*
+>    >
+>    > * Go on your raster data
+>    > * Click on {% icon galaxy-pencil %} to edit it
+>    > * Click on {% icon galaxy-chart-select-data %} Datatypes
+>    > * On "New Type" **Select** bil
+>    > * Press **Save**
+>    >
+>    >
+>    >    > <comment-title></comment-title>
+>    >    >
+>    >    > Do the same for the raster header with the datatype hdr
+>    >    {: .comment}
+>    >
+>    {: .tip}
+>
+>
+>    > <tip-title>You can use the output of **Compute a PCA**</tip-title>
+>    >
+>    > - {% icon param-select %} *"In which format are your data ?"*: 'Your already have the files in ENVI BIL format'
+>    > - {% icon param-file %} *"Input raster"*: `PCA raster` (output of **Compute a PCA** {% icon tool %})
+>    > - {% icon param-file %} *"Input header"*: `PCA header` (output of **Compute a PCA** {% icon tool %})
+>    > - {% icon param-text %} *"Write a number of the value of alpha"*: '1'
+>    >
+>    >
+>    > ***TODO***: *Here again check that the "Input raster" datatype is bil and that "Input raster header" datatype is hdr*
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+![Global biodiveristy tabular](../../images/remote_sensing/rasterdivtab.png "Global biodiveristy tabular")
+![Global biodiveristy graph](../../images/remote_sensing/rasterdiv.png "Global biodiveristy graph")
```

Comment:
oh, and could you update the alt texts (the bit between the `[..]` ) for the images to be a bit more descriptive? Imagine you are describing what is in the image to a blind person

Reply:
We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.